storage/oasis/nodeapi/file/kvstore.go

```diff
@@ -125,6 +125,7 @@ func glob(patterns []string, antipatterns []string) ([]string, error) {
 }
 
 // Moves all files that match the src glob patterns to the destination directory.
+// NOTE: If multiple source files have the same filename, one will clobber the others when moved!
```

Comment:
what's the thing about multiple source files with the same filename?

Reply:
update: it was if the input patterns would cover multiple dirs that contained files that had the same filename